PostgresAssignment/src/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "new",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
```

Comment:
You can add other scripts - https://github.com/adityajawade93/freshers-june2021/tree/main/tarun_kumar/PostgreAssignment